clap_complete/src/dynamic/completer.rs

```diff
@@ -304,15 +326,35 @@ fn complete_subcommand(value: &str, cmd: &clap::Command) -> Vec<CompletionCandid
 
 /// Gets all the long options, their visible aliases and flags of a [`clap::Command`].
 /// Includes `help` and `version` depending on the [`clap::Command`] settings.
-fn longs_and_visible_aliases(p: &clap::Command) -> Vec<(String, Option<StyledStr>)> {
+fn longs_and_visible_aliases(p: &clap::Command) -> Vec<CompletionCandidate> {
```

Comment:
Can you extract that to a separate refactor commit (or to the existing `CompletionCandidate` refactor)?